src/PendingCalls/TestCall.php

```diff
@@ -615,6 +622,20 @@ public function coversNothing(): self
         return $this;
     }
 
+    /**
+     * Adds a reference to the tested method or class. This helps to link test
+     * cases to the source code for easier navigation.
+     *
+     * @param  array<class-string|string>|class-string  ...$classes
+     */
+    public function reference(string|array ...$classes): self
```

Comment:
Perhaps `references()` to match the other methods. 👍🏻 I also think it makes more sense if used in a sentence, e.g.

> 'My test' references('a', 'b')

Reply:
this is a great idea! thank you